tests/unit/grpc_internals/zeebe_job_adapter_test.py

```diff
@@ -101,34 +102,34 @@ async def test_raises_on_already_completed_job(self, zeebe_adapter: ZeebeJobAdap
 @pytest.mark.asyncio
 class TestFailJob:
     async def test_response_is_of_correct_type(self, zeebe_adapter: ZeebeJobAdapter, first_active_job: Job):
-        response = await zeebe_adapter.fail_job(first_active_job.key, first_active_job.retries, random_message())
+        response = await zeebe_adapter.fail_job(first_active_job.key, first_active_job.retries, random_message(), 0, {})
```

Comment:
Create fixtures `retry_back_off_ms` and `variables` to make this a little cleaner

Reply:
Done